docs/inference/examples/bbh.rst

```diff
@@ -0,0 +1,207 @@
+duration is approximately determined by the total mass of a system. The lowest
+total mass (`= mass1 + mass2`) admitted by our prior is 20 solar masses. This
+corresponds to a duration of ~6 seconds, so we start the analysis time 6
+seconds before the trigger time. (See the :py:mod:`pycbc.waveform` module for
+utilities to estimate waveform duration.) Since the trigger time is
+approximately where we expect the merger to happen, we only need a small amount
+of time afterward to account for the timing uncertainty and ringdown. Here, we
+choose 2 seconds, which is a good safety margin.
+
+We also have to provide arguments for estimating a PSD. Although we know the
+exact shape of the PSD in this case, we will still estimate it from the
+generated data, as this most closely resembles what you do with a real event.
+To do this, we have set ``psd-estimation``  to ``median-mean`` and we have set
+``psd-segment-length``, ``psd-segment-stride``, and ``psd-(start|end)-time``
+(which are with respect to the trigger time). This means that a Welch-like
+method will be used to estimate the PSD. Specifically, we will use 512s of data
+centered on the trigger time to estimate the PSD. This time will be divided up
+into 8s-long segments (the segment length) each overlapping by 4s (the segment
+stride). The data in each segment will be transformed to the frequency domain.
+Two median values will be determined in each frequency bin from across all
+even/odd segments, then averaged to obtain the PSD.
+
+The beginning and end of the analysis segment will be corrupted by the
+convolution of the inverse PSD with the data. To limit the amount of time that
+is corrupted, we set ``psd-inverse-length`` to ``8``. This limits the
+corruption to at most the first and last four seconds of the data segment.  To
+account for the corruption, ``psd-inverse-length/2`` seconds are
+subtracted/added by the code from/to the analysis start/end times specified by
+the user before the data are transformed to the frequency domain.
+Consequently, our data will have a frequency resolution of :math:`\Delta f =
+1/16\,` Hz. The 4s at the beginning/end of the segment are effectively ignored
+since the waveform is contained entirely in the -6/+2s we set with the analysis
+start/end time.
+
+Finally, we will use the ``emcee_pt`` sampler with the following settings:
+
+.. literalinclude:: ../../../examples/inference/samplers/emcee_pt-gw150914_like.ini
+   :language: ini
+
+:download:`Download <../../../examples/inference/samplers/emcee_pt-gw150914_like.ini>`
+
+Here, we will use 200 walkers and 20 temperatures. We will checkpoint (i.e.,
+dump results to file) every 2000 iterations. Since we have provided an
+``effective-nsamples`` argument and a ``[sampler-burn_in]`` section,
+``pycbc_inference`` will run until it has acquired 1000 independent samples
+after burn-in, which is determined by a combination of the :py:meth:`nacl
+<pycbc.inference.burn_in.MultiTemperedMCMCBurnInTests.nacl>` and
```

Comment:
Change this to `<pycbc.inference.burn_in.MCMCBurnInTests.nacl>` to fix the cross link (sorry, this was a mistake I introduced when I wrote this).